src/category_theory/triangulated/basic.lean

```diff
@@ -48,19 +47,14 @@ structure triangle :=
 
 local attribute [instance] has_zero_object.has_zero
 instance [has_zero_object C] : inhabited (triangle C) :=
-⟨{ obj1 := 0,
-  obj2 := 0,
-  obj3 := 0,
-  mor1 := 0,
-  mor2 := 0,
-  mor3 := 0 }⟩
+⟨⟨0,0,0,0,0,0⟩⟩
 
 variable {C}
 
 /--
-A morphism of triangles (X,Y,Z,f,g,h)→(X',Y',Z',f',g',h') in C is a triple of morphisms
-a: X → X', b: Y → Y', c: Z → Z' such that b ≫ f = f' ≫ a, c ≫ g = g' ≫ b,
-and Σa ≫ h = h' ≫ c.
+A morphism of triangles `(X,Y,Z,f,g,h) ⟶ (X',Y',Z',f',g',h')` in `C` is a triple of morphisms
+`a : X ⟶ X'`, `b : Y ⟶ Y'`, `c : Z ⟶ Z'` such that
+`b ≫ f = f' ≫ a`, `c ≫ g = g' ≫ b`, and `a⟦1⟧' ≫ h = h' ≫ c`.
```

Comment:
These equations in the doc-string don't actually match with the axiom below. @l-kershaw, could you please fix them?

Reply:
(Feel free to push to this branch directly.)